MyLibrary/Sources/SharedModels/Favorites.swift

```diff
@@ -0,0 +1,30 @@
+public struct Favorites: Equatable, Codable {
+  private var eachConferenceFavorites: [String: [Session]]
+
+  public init(eachConferenceFavorites: [(conference: Conference, favoriteSessions: [Session])]) {
```

Comment:
Since initialized in second line, first parameter should be removed.

Reply:
Sorry for my code readability, but first parameter is necessary.

To create `Favorites`, give pairs which has conference and its favorites sessions.
Then convert given pairs to dictionary of [conference title: its favorites sessions], and set the dictionary to property.

I intentionally use conference as input to avoid using non-existent conference title as key.
I'll continue to think readable name.